src/main/java/com/dumbdogdiner/stickyapi/bukkit/particle/shapes/OutlinedCube.java

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright (c) 2020 DumbDogDiner <a href="dumbdogdiner.com">&lt;dumbdogdiner.com&gt;</a>. All rights reserved.
+ * Licensed under the MIT license, see LICENSE for more information...
+ */
+package com.dumbdogdiner.stickyapi.bukkit.particle.shapes;
+
+import com.dumbdogdiner.stickyapi.bukkit.particle.ParticleSystem;
+import com.dumbdogdiner.stickyapi.bukkit.particle.Shape;
+import javafx.geometry.Point3D;
+import lombok.Getter;
+import org.bukkit.Particle;
+
+import java.util.*;
+
+
+public class OutlinedCube implements Shape {
+    @Getter
+    private final double inscribedRadius;
+    @Getter
+    private final Point3D center;
+
+
+    /**
+     * Draws a cube out of particles
+     * @param center the center of the cube
+     * @param inscribedRadius the radius of a sphere inscribed in the cube
+     */
+    public OutlinedCube(Point3D center, double inscribedRadius){
+        this.center = center;
+        this.inscribedRadius = inscribedRadius;
```

Comment:
I doubt many people wod be working with inscribed radius - better to do side length.